internal/commands/download_sbom.go

```diff
@@ -0,0 +1,47 @@
+package commands
+
+import (
+	"github.com/pkg/errors"
+	"github.com/spf13/cobra"
+
+	cpkg "github.com/buildpacks/pack/pkg/client"
+	"github.com/buildpacks/pack/pkg/logging"
+)
+
+type DownloadSBOMFlags struct {
+	Local          bool
+	Remote         bool
+	DestinationDir string
+}
+
+func DownloadSBOM(
+	logger logging.Logger,
+	client PackClient,
+) *cobra.Command {
+	var flags DownloadSBOMFlags
+	cmd := &cobra.Command{
+		Use:     "download <image-name>",
+		Args:    cobra.ExactArgs(1),
+		Short:   "Download SBoM from specified image",
+		Long:    "Download layer containing Structured Bill of Materials (SBoM) from specified image",
+		Example: "pack sbom download buildpacksio/pack",
```

Comment:
Hopefully this will be more intuitive when `buildpacksio/pack` works. Currently, when I try, I get
```
> ./out/pack sbom download buildpacksio/pack --local --output-dir /tmp/pack-sbom
ERROR: could not find SBoM information on 'buildpacksio/pack'
```
but by the time we release this, we should have a new version of the image with sbom information